packages/ogg/ogg.0.7.4/opam

```diff
@@ -0,0 +1,38 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bindings to libogg"
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+license: "GPL-2.0"
```

Comment:
```suggestion
license: "GPL-2.0-only"
```

or 

```suggestion
license: "GPL-2.0-or-later"
```

Reply:
Thanks! It was actually the wrong license.. Fix pushed upstream.